MarkerInfo[] markers  # Array of detected markers